SpatialGDK/Source/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -280,14 +288,6 @@ FNetworkGUID FSpatialNetGUIDCache::GenerateNewNetGUID(const int32 IsStatic)
 FNetworkGUID FSpatialNetGUIDCache::GetOrAssignNetGUID_SpatialGDK(const UObject* Object)
 {
 	FNetworkGUID NetGUID = GetOrAssignNetGUID(Object);
-	if (Object != nullptr)
```

Comment:
No love for comments? Perhaps make it verbose and add it back in if we aren't in the editor.

Reply:
I think that log was kind of useless? The one below is more useful. Don't have a strong preference though.